paper-api/src/main/java/org/bukkit/block/Biome.java

```diff
@@ -198,6 +201,27 @@ static Biome[] values() {
     }
 
     // Paper start
+    /**
+     * Get the biome's special effects.
+     *
+     * @return the biome's special effects
+     */
+    @NotNull BiomeSpecialEffects specialEffects();
+
+    /**
+     * Get the biome's climate.
+     *
+     * @return the biome's climate
+     */
+    @NotNull BiomeClimate climate();
+
+    /**
+     * Mob spawn settings for this biome.
+     *
+     * @return the mob spawn settings
+     */
+    @NotNull BiomeMobSpawning mobSpawning();
+
```

Comment:
i feel here make more sense the use of "get"?

Reply:
Most getters here, like [translationKey](https://jd.papermc.io/paper/1.21.5/org/bukkit/block/Biome.html#translationKey())() or [key](https://jd.papermc.io/paper/1.21.5/org/bukkit/Keyed.html#key()), are already without the get prefix

Reply:
Because come from the adventure style (?

Maybe more comments (from others) about these... But feel strange this type of class with that style